lms/static/sass/discussion/_discussion.scss

```diff
@@ -324,27 +341,17 @@ body.discussion {
 }
 
 .discussion-show {
+  @include float(right);
   position: relative;
   top: 3px;
   font-size: $forum-base-font-size;
   text-align: center;
 
-  &.shown {
-    .show-hide-discussion-icon {
-      background-position: 0 0;
-    }
+  &.btn.btn-brand.shown {
```

Comment:
Does this mean it is only true for buttons that have both classes "btn" and "btn-brand"? Since this is already only true for elements with class "discussion-show", I'd change this to be just be:

    &.shown {